src/mbgl/renderer/sources/render_raster_dem_source.hpp

```diff
@@ -6,6 +6,38 @@
 
 namespace mbgl {
 
+enum class DEMTileNeighbors {
```

Comment:
The underlying type of a C++ `enum` is implementation-defined. It may not choose the smallest integral type to fit all the values. We can be more specific by providing a type `enum class DEMTileNeighbors: unsigned char`